test/Groups/products.jl

```diff
@@ -1,3 +1,88 @@
+Oscar.@_AuxDocTest "show and print group products", (fix = false),
+raw"""
+common setup
+
+```jldoctest group.products.show
+julia> using Oscar
+
+julia> G = cyclic_group(3);
+
+julia> H = symmetric_group(3);
+
+julia> A = automorphism_group(G);
+
+julia> D = direct_product(G, H);
+
+julia> S = semidirect_product(G, id_hom(A), A);
+
+julia> W = wreath_product(G, H);
+```
+
+default `show` without unicode
+
+```jldoctest group.products.show
+julia> old = allow_unicode(false; temporary=true);
+
+julia> show(stdout, D)
+pc group x Sym(3)
+
+julia> show(stdout, MIME("text/plain"), D)
+Direct product of
+  Pc group of order 3
+  Sym(3)
+
+julia> show(stdout, S)
+pc group : aut( <pc group of size 3 with 1 generator> )
+
+julia> show(stdout, MIME("text/plain"), S)
+Semidirect product of
+  pc group of order 3
+  aut( <pc group of size 3 with 1 generator> )
+
+julia> show(stdout, W)
+pc group wr Sym(3)
```

Comment:
```suggestion
Pc group wr Sym(3)
```
the first letter in this cases should be uppercase